fs/fs.go

```diff
@@ -222,6 +223,10 @@ func processMounts(mounts []*mount.Info, excludedMountpointPrefixes []string) ma
 			}
 		}
 
+		if mount.Fstype == "overlay" {
+			mount.Source = fmt.Sprintf("%s_%d-%d", mount.Source, mount.Major, mount.Minor)
```

Comment:
why are we setting source here?

Reply:
When mount.Source is not unique to overlayfs mount in the mountinfo file, the mount.Source is "overlay" for all overlayfs mounts, it causes problems when it is used as key in the partitions map. Hence this is a way to make mount.Source unique.

Reply:
can you add a comment explaining that this makes makes mount.Source unique for overlay mounts?